integration/Earthfile

```diff
@@ -1,35 +1,66 @@
-FROM hseeberger/scala-sbt:11.0.6_1.3.10_2.13.1
+FROM docker:19.03.7-dind
 WORKDIR /scala-example
+RUN apk add openjdk11 curl docker-compose bash
+
+#Scala
+# Defaults if not specified in --build-arg
+ARG sbt_version=1.3.2
+ARG sbt_home=/usr/local/sbt
+
+# Download and extract from archive
+RUN apk add --no-cache --update wget
```

Comment:
Perhaps `wget` should be installed at the top, in the other `apk` command?

Reply:
Will do